pyaerocom/aeroval/setupclasses.py

```diff
@@ -301,15 +303,19 @@ class EvalSetup(BaseModel):
     ########################################
 
     IGNORE_JSON: list[str] = ["_aux_funs"]
-    ADD_GLOB: list[str] = ["io_aux_file"]
+    ADD_GLOB: list[str] = ["io_aux_file", "var_web_info_file", "var_scale_colmap_file"]
```

Comment:
ADD_GLOB and IGNORE_JSON are attributes of BrowseDict - they have absolute no meaning for BaseModels.
_aux_funs is removed from json because BaseModel ignores fields starting with _.

I will remove the no longer required fields which were a remainder from the time we used BrowseDict.